db/migrate/20161225120120_add_reporter_id_to_issue.rb

```diff
@@ -0,0 +1,5 @@
+class AddReporterIdToIssue < ActiveRecord::Migration[5.0]
+  def change
+    add_reference :issues, :user, foreign_key: true
```

Comment:
remove this foreign_key: true
I know it often in Rails we have issues because of it. Even if many recommends it - I would suggest to remove foreign_key but add index